UserInterface/ProgressBarDemos/ProgressBarDemos/ProgressBarDemos/ProgressBarCodePage.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using Xamarin.Forms;
+
+namespace ProgressBarDemos
+{
+    public class ProgressBarCodePage : ContentPage
+    {
+        ProgressBar defaultProgressBar;
+        ProgressBar styledProgressBar;
+        float progress = 0f;
+
+        public ProgressBarCodePage()
+        {
+            Title = "ProgressBar Code Demo";
+            Padding = 10;
+
+            defaultProgressBar = new ProgressBar
+            {
+                WidthRequest = 500,
+                HorizontalOptions = LayoutOptions.Fill,
+                VerticalOptions = LayoutOptions.CenterAndExpand
+            };
+
+            styledProgressBar = new ProgressBar
+            {
+                ProgressColor = Color.Orange,
+                WidthRequest = 500,
+                HorizontalOptions = LayoutOptions.Fill,
+                VerticalOptions = LayoutOptions.CenterAndExpand
+            };
+
+            var progressButton = new Button
+            {
+                Text = "Click to Progress",
+                HorizontalOptions = LayoutOptions.CenterAndExpand,
+                VerticalOptions = LayoutOptions.CenterAndExpand
+            };
+
+            progressButton.Clicked += Button_Clicked;
```

Comment:
Can we go with `OnButtonClicked` for all event handlers in this sample? It's another convention used throughout our samples - always On<Control><Event>.

Reply:
Per an earlier comment I added an `await` which also meant I added an `async` tag on the handler. So I renamed them `OnButtonClickedAsync`.

Reply:
generally the pattern is to only add "Async" if the method return a task, not void.  So event handlers usually return void.